website/src/components/SearchPage/DownloadDialog/DownloadForm.tsx

```diff
@@ -214,13 +235,22 @@ export const DownloadForm: FC<DownloadFormProps> = ({
                     onSelect={setIncludeRestricted}
                 />
             )}
-            <RadioOptionBlock
-                name='dataType'
-                title='Data type'
-                options={dataTypeOptions}
-                selected={dataType}
-                onSelect={setDataType}
-            />
+            <div className='flex-1 min-w-0'>
+                <RadioOptionBlock
+                    name='dataType'
+                    title='Data type'
+                    options={getDataTypeOptions()}
+                    selected={dataType}
+                    onSelect={setDataType}
+                />
+                {disableAlignedSequences && suborganismIdentifierField !== undefined && (
+                    <div className='text-sm text-gray-400 mt-4 max-w-60'>
+                        Or select a {formatLabel(suborganismIdentifierField)} with the search UI to enable download of
```

Comment:
sorry a bit delayed could we remove the `Or` here?

Reply:
The "Or" was a considered choice to make clear that this is a distinct option to those above but ofc it's not a big deal either way